src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -3781,10 +3781,12 @@ private static void GetSymbolsAndResultKind(BoundUnaryOperator unaryOperator, ou
             {
                 Debug.Assert((object)unaryOperator.MethodOpt == null && unaryOperator.OriginalUserDefinedOperatorsOpt.IsDefaultOrEmpty);
                 UnaryOperatorKind op = unaryOperator.OperatorKind.Operator();
-                symbols = ImmutableArray.Create<Symbol>(new SynthesizedIntrinsicOperatorSymbol(unaryOperator.Operand.Type.StrippedType(),
-                                                                                                 OperatorFacts.UnaryOperatorNameFromOperatorKind(op, isChecked: unaryOperator.OperatorKind.IsChecked()),
-                                                                                                 unaryOperator.Type.StrippedType(),
-                                                                                                 unaryOperator.OperatorKind.IsChecked()));
+                bool isChecked = unaryOperator.OperatorKind.IsChecked();
```

Comment:
>bool isChecked = unaryOperator.OperatorKind.IsChecked(); [](http://example.com/codeflow?start=16&length=56)

Please revert the refactoring. I am assuming we don't want to make any behavior changes here

Reply:
Applies to other changes in this file.

Reply:
it's not really a refactoring.  It's more about ensuring i don't accidentally pass the wrong value into one of these.  :)

Reply:
I do not think we want to make any changes here for the purpose of this PR.

Reply:
We want to clearly see that nothing is changed here.

Reply:
ok.  reverted.